src/frontend/qt_sdl/main.cpp

```diff
@@ -3027,7 +3077,9 @@ void MainWindow::onCameraSettingsFinished(int res)
 
 void MainWindow::onOpenAudioSettings()
 {
-    AudioSettingsDialog* dlg = AudioSettingsDialog::openDlg(this);
+    AudioSettingsDialog* dlg = AudioSettingsDialog::openDlg(this, emuThread->emuIsActive());
+    connect(emuThread, &EmuThread::syncVolumeLevel, dlg, AudioSettingsDialog::onSyncVolumeLevel);
```

Comment:
I think you need to explicitly take the address of onSyncVolumeLevel here for the error to go away, though I'm not sure.